compiler/luci/import/src/Nodes/CircleReshape.cpp

```diff
@@ -30,17 +30,16 @@ bool CircleReshapeGraphBuilder::validate(const ValidateArgs &args) const
   if (args.op.outputs.size() != 1)
     return false;
 
-  // for two inputs, check if type is S32
+  // for two inputs, check if type is S32 or S64
   if (args.op.inputs.size() == 2)
   {
     const auto &inputs = args.op.inputs;
     const auto tensors = args.reader.tensors();
     const auto tensor_in = tensors.at(inputs.at(1));
     assert(tensor_in != nullptr);
 
-    // NOTE fix this if there is any other case
-    // TensorFlow lite and circle only supports S32
```

Comment:
existing comment `only supports S32` is for attribute not second input. so strictly speaking, this comment is incorrect.